### PR TITLE
Add more expected stub behaviour

### PR DESCRIPTION
**Pull request by DanielBrookRoberge**
This PR adds more documentation to the stub connector, and implements a few more behaviours.

 **JIRA**: https://mobify.atlassian.net/browse/WEBDATA-18

## Changes
- Log the arguments to the stub commands, as an implementation aid
- Fix inconsistency between IM definition of `fetchShippingMethodsEstimate` and connector definitions
- Make the login page do the right thing if it doesn't get an URL from the connector.

## How to test-drive this PR
- See if the new source code is more explanatory.

### CODE REVIEW

File: web/app/integration-manager/_stub-connector/checkout/commands.js [hunk 1]
```diff
@@ -5,32 +5,34 @@
 /* eslint-disable no-unused-vars */
 
 export const initCheckoutShippingPage = (url) => (dispatch) => {
-    console.log('[Stub Connector] Called initCheckoutShippingPage stub')
+    console.log('[Stub Connector] Called initCheckoutShippingPage stub with arguments', url)
```
Comment (jnurse) [inline web/app/integration-manager/_stub-connector/checkout/commands.js]: All init__Page commands receive a routeName as well.

File: web/app/integration-manager/_stub-connector/checkout/commands.js [hunk 2]
```diff
@@ -5,32 +5,34 @@
...
     return Promise.resolve()
 }
 
 export const initCheckoutPaymentPage = (url) => (dispatch) => {
-    console.log('[Stub Connector] Called initCheckoutPaymentPage stub')
+    console.log('[Stub Connector] Called initCheckoutPaymentPage stub with arguments', url)
```
Comment (jnurse) [inline web/app/integration-manager/_stub-connector/checkout/commands.js]: [Nit-pick/non-blocking] In the cart and account command files you had a colon after arguments.

Comment (DanielBrookRoberge) [inline web/app/integration-manager/_stub-connector/checkout/commands.js]: Fixed!

File: web/app/integration-manager/_stub-connector/checkout/commands.js (hunk 1 above)
Comment (DanielBrookRoberge) [inline web/app/integration-manager/_stub-connector/checkout/commands.js]: Fixed!

File: web/app/integration-manager/_stub-connector/account/commands.js [hunk 3]
```diff
@@ -2,44 +2,54 @@
...
 export const logout = () => (dispatch) => {
     console.log('[Stub Connector] Called logout stub')
+    dispatch(setLoggedIn(false))
     return Promise.resolve()
 }
 
 export const registerUser = (firstname, lastname, email, password) => (dispatch) => {
-    console.log('[Stub Connector] Called registerUser stub')
+    console.log('[Stub Connector] Called registerUser stub with parameters:', firstname, lastname, email, password)
     return Promise.resolve()
```
Comment (jnurse) [inline web/app/integration-manager/_stub-connector/account/commands.js]: registerUser should also return a URL of the account page.

Comment (DanielBrookRoberge) [inline web/app/integration-manager/_stub-connector/account/commands.js]: Good catch, thanks!

**Pull request merged**